### DISCUSSION TITLE
Возврат кнопки «Назад» в режиме Моя волна

### DISCUSSION THREAD

**Discussion by VladWinner** [Chimildic/YaMuTools#51] [Идея] [answered]
Знаю, что в веб-версии нельзя проматывать на предыдущий трек в режиме прослушивания Моей волны, и кнопка при этом скрывается, но есть предложение: 
1) отменить её сокрытие в стилях
за это отвечает данный пункт:
![image](https://user-images.githubusercontent.com/13555921/165401184-4b2565bc-d224-4aad-b40a-cdead1a2a451.png)
2) в этом режиме заменять её цвет на красный или более прозрачный, чтобы было понятно, что она может промотать текущий трек в начало, но не переключить на предыдущий
По крайней мере, это избавит от необходимости целиться курсором для повторного прослушивания трека. Не полное возвращение функциональности, но всяко лучше, чем сейчас.

**Comment by Chimildic** [comment, marked as answer]
Расширение добавляет кнопку репита для потоков. В таком случае текущий трек начнется с начала.
По сути, происходит тоже самое, что вы описали. Разве что нужно отключить репит, чтобы поток продолжился. 

Возможно, стоит предусмотреть режим, при котором репит отключается сам. Вместо махинаций с цветами у другой кнопки.
